CONTRIBUTING.md

```diff
@@ -100,9 +100,17 @@ Code
  - 80 character line length
  - commas last
  - declare variables in the outermost scope that they are used
+ - camelCase for variables in JavaScript
+ - snake_case for variables in Python
 
 In general, try to make your code blend in with the surrounding code.
 
+## Pull Requests
+ 
+ - link back to the original issue(s) whenever possible
+ - commits should be squash before merging
+ - large commits should be broken into multiple logically cohesive commits if possible
```

Comment:
I think either works.  There was that one pull request that you split into two commits.  That worked well, but people may want to submit separate requests.